data.sql

```diff
@@ -5,3 +5,13 @@ VALUES (1, 'Agumon', '2020-02-03', 0, true, 10.23),
        (2, 'Gabumon', '2018-11-15', 2, true, 8),
        (3, 'Pikachu', '2021-01-07', 1, false, 15.04),
        (4, 'Devimon', '2017-05-12', 5, true, 11);
+
+--  FOR 2nd Part
+
+INSERT INTO animals VALUES (1,' Charmander', '2020-02-08', 0, false, 11);
+INSERT INTO animals VALUES (2,' Plantmon', '2021-11-15', 2, true, 5.7);
+INSERT INTO animals VALUES (3,' Squirtle', '1993-04-02', 3, false, 12.13);
+INSERT INTO animals VALUES (4,' Angemon', '2005-06-12', 1, true, 45);
```

Comment:
- [ ] Great job so far, 👏🏻  However, you need to pay more attention to the project requirements, actually the fourth first insert should have `weight_kg` with a negative value. kindly correct this 👍🏻  Your code should look like this:

```sql
    INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
    VALUES ('Charmander', '2020/2/8', '0', 'false', '-11.0');

    INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
    VALUES ('Plantmon', '2021/11/15', '2', 'true', '-5.7');

    INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
    VALUES ('Squirtle', '1993/04/02', '3', 'false', '-12.13');

    INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
    VALUES ('Angemon', '2005/06/12', '1', 'true', '-45.0');
```